src/database/client.rs

```diff
@@ -327,4 +328,27 @@ impl PgDbClient {
     ) -> anyhow::Result<Vec<PullRequestModel>> {
         get_merge_queue_prs(&self.pool, repo, tree_priority.map(|p| p as i32)).await
     }
+
+    pub async fn get_comments(
```

Comment:
```suggestion
    pub async fn get_tagged_bot_comments(
```
Just to make it more obvious that it's just (some, tagged) comments made by the bot.